astro-client-platform-core/client.go

```diff
@@ -24,10 +28,31 @@ type CoreClient = ClientWithResponsesInterface
 // create api client for astro platform core services
 func NewPlatformCoreClient(c *httputil.HTTPClient) *ClientWithResponses {
 	// we append base url in request editor, so set to an empty string here
-	cl, _ := NewClientWithResponses("", WithHTTPClient(c.HTTPClient), WithRequestEditorFn(astrocore.CoreRequestEditor))
+	cl, _ := NewClientWithResponses("", WithHTTPClient(c.HTTPClient), WithRequestEditorFn(requestEditor))
 	return cl
 }
 
+func requestEditor(ctx httpContext.Context, req *http.Request) error {
```

Comment:
This functions seem repetitive for all clients we generate? Should we make a common helper and pass api version to it?